CLAUDE.md

```
@@ -0,0 +1,621 @@
+所有样式都应使用主题，不允许存在硬编码颜色、字号等。
+
+# ReadeckApp Flutter 工程架构规范文档
+
+基于 Flutter 官方推荐的架构模式和最佳实践
+
+## 目录
+
+1. [项目概述](#项目概述)
+2. [核心架构原则](#核心架构原则)
+3. [分层架构设计](#分层架构设计)
+4. [MVVM 架构模式](#mvvm架构模式)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)